<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#34495e;
}
</string>
  </property>
  <widget class="QTextEdit" name="window_s">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>731</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color:none;
border-style:solid;
border-width: 3px;
border-radius: 10px;
border-color: #3498db;
color:black;
}
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="main_m">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>В главное меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Словарь программы </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
